api/v1alpha1/contour_types.go

```diff
@@ -49,13 +49,13 @@ type ContourSpec struct {
 	//
 	// +kubebuilder:default=2
 	// +kubebuilder:validation:Minimum=0
-	Replicas int32 `json:"replicas"`
+	Replicas int32 `json:"replicas,omitempty"`
```

Comment:
I thought this meant to be required from a previous different PR?

Reply:
@stevesloka the field is required in a sense. It's not required for a user to set, but it's required when the object is persisted. For example, a user can create a contour object with:
```
apiVersion: operator.projectcontour.io/v1alpha1
kind: Contour
metadata:
  name: contour-sample
spec: {}
```

which will be realized (thanks for kubebuilder defaulting) to:
```
apiVersion: operator.projectcontour.io/v1alpha1
kind: Contour
metadata:
  name: contour-sample
spec:
  replicas: 2
  namespace:
    name: projectcontour
    removeOnDeletion: false
status:
<SNIP>
```